fairlearn/metrics/_metric_frame.py

```diff
@@ -29,6 +29,9 @@
 _TOO_MANY_FEATURE_DIMS = "Feature array has too many dimensions"
 _SAMPLE_PARAM_KEYS_NOT_IN_FUNC_DICT = \
     "Keys in 'sample_params' do not match those in 'metric'"
+_INVALID_ERRORS_VALUE_ERROR_MESSAGE = "Invalid error value specified"
```

Comment:
Please specify the valid values - it might be helpful to define another constant along the lines of:
```python
_VALID_ERROR_STRINGS = [ 'raise', 'coerce' ]
```
and reference that in strings where needed (and in the code where the check is made).